cs/index.json

```diff
@@ -55,7 +56,11 @@
     "general" : "Obecné",
     "mining": "Těžba",
     "about" : "O programu",
-    "log_to_file" : "Protokolovat do souboru"
+    "log_to_file" : "Protokolovat do souboru",
+    "mining_algorithms": "Mining algorithms",
+    "mining_algorithms_explaination": "Select algorithm(s) that you want to mine",
+    "mining_algorithms_warning": "Please select at least one algorithm",
+    "mining_algorithms_updated": "Mining algorithms updated successfully"
   },
```

Comment:
```
"update": "Aktualizovat"
"mining_algorithms": "Těžební algoritmy",
"mining_algorithms_explaination": "Vyberte algoritmus/algoritmy pro těžbu",
"mining_algorithms_warning": "Vyberte alespoň jeden algoritmus",
"mining_algorithms_updated": "Těžební algoritmy úspěšně aktualizovány"
```

Reply:
Key was changed to preferred, Please help to update it if you have time, Thanks

Reply:
It can stay as it is, but after change, `"Preferované algoritmy"` would be more accurate